src/init.cpp

```diff
@@ -2010,6 +2010,11 @@ bool AppInit2()
             // Run a thread to precompute any zPIV spends
             threadGroup.create_thread(boost::bind(&ThreadPrecomputeSpends));
         }
+
+        if (GetBoolArg("-staking", true) && pwalletMain) {
```

Comment:
minor nit.  pwalletMain check isn't necessary since you're contained within `if (pwalletMain) {}`

Reply:
fixed